package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "vite-plugin-vue2",
-  "version": "1.9.0",
```

Comment:
Can you restore this changes?

Reply:
@antfu 
Hello there, I did revert commit and there's only necessary file changes now. Please review again thx.